andela/krypton-ah-frontend
# #164171687 Integrate user authentication components with the logic (Signup)

#### What does this PR do?
Integrate the signup form component with the user authentication actions and reducers
#### Description of Task to be completed?
To link the signup frontend to the backend
#### How should this be manually tested?
Git clone the repo and cd into it
npm run dev to start the server
click on write article on the homepage
click on signup on the modal that appears
fill in the signup information and click enter
open the redux dev tools to check the application state
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#164171687
#### Screenshots (if appropriate)
<img width="1003" alt="screenshot 2019-02-22 at 4 34 18 pm" src="https://user-images.githubusercontent.com/20207295/53253484-abc65280-36c1-11e9-8e90-54f790ecfc2a.png">
<img width="1108" alt="screenshot 2019-02-22 at 4 37 19 pm" src="https://user-images.githubusercontent.com/20207295/53253494-aff27000-36c1-11e9-8255-fecbc0962b12.png">
<img width="1291" alt="screenshot 2019-02-22 at 4 37 41 pm" src="https://user-images.githubusercontent.com/20207295/53253501-b54fba80-36c1-11e9-9d5e-3391473e8c30.png">
<img width="588" alt="screenshot 2019-02-22 at 4 38 00 pm" src="https://user-images.githubusercontent.com/20207295/53253507-b7b21480-36c1-11e9-8100-13a867a25152.png">
<img width="988" alt="screenshot 2019-02-22 at 4 38 24 pm" src="https://user-images.githubusercontent.com/20207295/53253513-bb459b80-36c1-11e9-97e9-23b4a4edc684.png">

#### Questions:
